benchbuild/project.py

```diff
@@ -227,16 +216,16 @@
         return uuid.uuid4()
 
     @run_uuid.validator
-    def __check_if_uuid(self, _, value):
+    def __check_if_uuid(self, _, value) -> None:
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)